server/src/main/java/org/apache/uniffle/server/RegisterHeartBeat.java

```diff
@@ -137,7 +140,9 @@ boolean sendHeartBeat(
               sendSuccessfully.set(true);
             }
           } catch (Exception e) {
-            LOG.error(e.getMessage());
+            if (!shutdown) {
+              LOG.error(e.getMessage());
```

Comment:
I think `LOG.error("xxx", e);` is better than `LOG.error(e.getMessage());`. Because it will give a full stack trace.